libs/sdk-ui-tests-e2e/cypress/tools/drillToModal.ts

```diff
@@ -73,12 +73,36 @@ export class DrillToModal {
             .find(".s-drill-down")
             .should("be.visible")
             .contains(`${attr}`)
-            .click();
+            .click({ force: true });
         return this;
     }
 
     hasTitleHeader(title: string) {
         this.getTitleElement().should("have.text", title);
         return this;
     }
+
+    getTooltipContentsModal(index: number) {
```

Comment:
no need this fuction, you can use getChart() then reuse the function in chart.ts